src/include/miopen/tensor.hpp

```diff
@@ -285,7 +285,7 @@ struct TensorDescriptor : miopenTensorDescriptor
 };
 
 template <class TElement>
-constexpr auto GetNCDHW(int spatial_dims, const std::vector<TElement>& data)
+constexpr auto GetNCDHW(unsigned spatial_dims, const std::vector<TElement>& data)
```

Comment:
[Q] why this is necessary?

Reply:
This variable is always positive and using a signed data type can be misleading

Reply:
That is, you just caught a not quite high-quality code and decided to fix it, right? 😄

Reply:
Yes, you are right :smile:. I decided to change this too, because it is related to my changes